backend/prometheus.yml

```
@@ -0,0 +1,12 @@
+global:
+  scrape_interval: 5s
+
+scrape_configs:
+  - job_name: 'backend'
+    metrics_path: '/metrics'
+    static_configs:
+      - targets: ['backend:8000']
+
+  - job_name: 'node-exporter'
+    static_configs:
+      - targets: ['node-exporter:9100']
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)